python/geocode.py

```diff
@@ -103,7 +104,9 @@ def parse_output(geo):
                 streetname as street_name, 
                 borough,
                 'bis' as source
-            FROM dob_jobapplications UNION
+            FROM dob_jobapplications 
+            where gid::text = '1'
```

Comment:
I think it would be useful to include a comment as to why you are filtering the gid here and what impact that has on geocode step.

Reply:
Essentially you are reducing the number of records to geocode but why on records where gid = '1'?

Reply:
I agree adding a comment would be useful.  This relates to [this script](https://github.com/NYCPlanning/db-developments/blob/74d04c2667a8519cece5b59e324ebcd27c857c8e/templates/dob_jobapplications.yml) and [this script](https://github.com/NYCPlanning/db-developments/blob/74d04c2667a8519cece5b59e324ebcd27c857c8e/sql/bis/_init.sql), and see [this PR](https://github.com/NYCPlanning/db-developments/pull/503/files) for an explanation.